sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Generated/Models/IPv6FirewallRuleList.cs

```diff
@@ -1,51 +0,0 @@
-// <auto-generated>
```

Comment:
Introduced with https://github.com/Azure/azure-rest-api-specs/pull/18490 @LeiWang3 can you confirm this is not a breaking change (and other IPv6 related changes in this pull request)